modules/layers/src/solid-polygon-layer/polygon-tesselator.js

```diff
@@ -70,9 +70,9 @@ export default class PolygonTesselator extends Tesselator {
           target,
           size: 4,
           getValue: object => {
-            const color = accessor(object);
-            if (isNaN(color[3])) {
-              color[3] = 255;
+            let color = accessor(object);
+            if (color.length < 4) {
```

Comment:
Is it worth warning when the accessor returns a color that is not a vec3 or vec4? This is one of the issues we ran into with the bug described in the issue linked to this PR.

Reply:
Yeah that's a good point. @Pessimistress's suggestion ensures that we always return a vec4, which handles incorrect inputs better than my initial fix.